code/game/machinery/newscaster.dm

```diff
@@ -283,9 +288,7 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 		return 1
 
 	if(href_list["set_channel_name"])
-		channel_name = sanitizeSQL(strip_html_simple(input(usr, "Provide a Feed Channel Name", "Network Channel Handler", "")))
-		while(findtext(channel_name," ") == 1)
-			channel_name = copytext(channel_name, 2, lentext(channel_name) + 1)
+		channel_name = trim(sanitizeSQL(strip_html_simple(input(usr, "Provide a Feed Channel Name", "Network Channel Handler", ""))))
```

Comment:
@tigercat2000 or @Fox-McCloud is `sanitizeSQL` here strictly necessary? It makes it so you can't use quotes in the feed channel, which is a bit annoying (I disabled sanitizeSQL when testing this PR). I don't think this is going into a SQL database, is it?